actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   AEAD encrypted cookies and sessions
+
+    Encrypted cookies may now use AES in GCM mode. To have existing cookies be
+    seamlessly upgraded to this new scheme set the
+    `action_dispatch.use_authenticated_cookie_encryption` configuration value to
+    `true`. This value can be set 
```

Comment:
Let's mention the AES GCM benefits here (faster, shorter — copy from the security guide if you like) and that `new_framework_defaults_5.2.rb` shows how to upgrade.